.github/workflows/ansible-release.yml

```diff
@@ -49,10 +47,39 @@ jobs:
       - name: Install dependencies
         working-directory: antsibull
         run: |
+          python3 -m pip install packaging
           python3 -m pip install ansible-core
           python3 -m pip install antsibull
           ansible-galaxy install -r requirements.yml
 
+      - name: Validate version and extract major version
+        shell: python
+        id: extract-version
+        run: |
+          import os
+          import pathlib
+          import sys
+
+          from packaging.version import Version
+
+          FILE_APPEND_MODE = 'a'
```

Comment:
Does this really need to be a constant?

Reply:
I copied this and the below from https://github.com/ansible-community/ansible-test-gh-action, it seems to be @webknjaz's preferred way of outputting things from a Python script in GHA. Maybe he can comment on it :)

Reply:
Okay, fair enough

Reply:
Yeah, I like to name the data. Otherwise, "what's is that weird `'a'` string constant being passed to a function as a positional argument?" adds some cognitive load for people that are out of context. Some day, I'll move most of that to an actual Python file and this will reduce the boilerplate too..